cirq-ft/cirq_ft/algos/and_gate.ipynb

```diff
@@ -111,12 +111,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import numpy as np\n",
+    "\n",
     "input_states = [(a, b, 0) for a, b in itertools.product([0, 1], repeat=2)]\n",
     "output_states = [(a, b, a & b) for a, b, _ in input_states]\n",
     "\n",
     "\n",
     "for inp, out in zip(input_states, output_states):\n",
-    "    result = cirq.Simulator().simulate(c2, initial_state=inp)\n",
+    "    result = cirq.Simulator(dtype=np.complex128).simulate(c2, initial_state=inp)\n",
```

Comment:
why this change?

Reply:
Thet test started failing for Windows because of precision issues. I didn't dig very deep on whether it's something related to the new release or whether the test was flaky in the first place (probably more likely) so I just increased the precision and it seems to work fine.